app/screens/home/search/recent_searches/recent_searches.tsx

```diff
@@ -54,6 +55,7 @@ const RecentSearches = ({setRecentValue, recentSearches, teamName}: Props) => {
     const renderRecentItem = useCallback(({item}) => {
         return (
             <RecentItem
+                onRemoveSearch={onRemoveSearch}
```

Comment:
Should this go to the dependencies of renderRecentItem?

Reply:
this has been removed after refactoring